docs/design-docs/benchmarks.md

```diff
@@ -181,7 +181,7 @@ This chapter will describe the interfaces with the caller (Superbench executor),
 
 The inputs needed by the `benchmarks` package is simple, just the context object of the benchmark want to run:
 
-### Invoke
+#### Invoke
```

Comment:
Why change this?

Reply:
It should be `### Inputs`'s `#### Invoke`, just align with `### Outputs`

Reply:
Got it!